king_phisher/client/tabs/mail.py

```diff
@@ -847,6 +847,37 @@ def signal_button_clicked_verify(self, button):
 			gui_utilities.show_dialog_info('Successfully Opened The Web Server URL', self.parent)
 		return
 
+	def signal_button_clicked_verify_spf(self, button):
+		sender_email = self.gobjects['entry_source_email_smtp'].get_text()
+		error_description = None
```

Comment:
I meant the `error_description` variable, is that one used? I had originally reviewed this on my cell so it wasn't the easiest way to review things.